speedy-recovery-client/src/components/secured/home/HomePageForParent.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class HomePageForParent extends Component {
+  static defaultProps = {
+    event: {
+      start: [],
+      patient: [],
+      user: []
+    }
+  };
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      appointmentTime: {},
+      user: {},
+      patient: {}
+    };
+  }
+
+  componentWillMount() {
+    setTimeout(() => {
+      this.setState({
+        appointmentTime: this.props.event.start,
+        user: this.props.user,
+        patient: this.props.event.patient
+      });
+    }, 1000);
+  }
+  render() {
+    return (
+      <div>
+        <h3>Hello {this.state.user.name}</h3>
+        {new Date(this.state.appointmentTime).toLocaleString("en-uk") !==
+        "Invalid Date" ? (
+          <h4>
```

Comment:
Please use `<p`>, this is not a heading

Reply:
(of course my comments here also refer to the equivalent parts of the other roles)